scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html

```diff
@@ -184,7 +184,7 @@ <h5>
       <div class="col-sm-7 text-center">
         <input type="file" name="csv_file" class="custom-file-input" required onchange="this.nextElementSibling.innerText = this.files[0].name">
         <label class="custom-file-label" for="csv_file">Choose file</label><br>
-        <p class="help-block">How do I format my <a href="https://clinical-genomics.github.io/scout/user-guide/variants/#Managed-variants-upload-file-format" rel="noopener" target="_blank">managed variants file</a>?</p>
+        <p class="help-block">How do I format my <a href="https://clinical-genomics.github.io/scout/user-guide/variants/#managed-variants-upload-file-format" rel="noopener" target="_blank">managed variants file</a>?</p>
```

Comment:
The current link doesn't work

Reply:
Nice catch! It went to the top of the page instead..